contrib/guix/manifest.scm

```diff
@@ -87,9 +92,9 @@ http://www.linuxfromscratch.org/hlfs/view/development/chapter05/gcc-pass1.html"
   (let* ((xbinutils (cross-binutils target))
          ;; 1. Build a cross-compiling gcc without targeting any libc, derived
          ;; from BASE-GCC-FOR-LIBC
-         (xgcc-sans-libc (cross-gcc target
+         (xgcc-sans-libc (cross-gcc-with-newlib (cross-gcc target
```

Comment:
Our goal, as it's been stated in the comment above, is to
> Build a cross-compiling gcc without targeting any libc

But [`newlib`](https://www.sourceware.org/newlib/) _is_ a libc implementation.